buildout.cfg

```diff
@@ -1,12 +1,12 @@
 [buildout]
 parts = test
-extends = https://raw.githubusercontent.com/zopefoundation/Zope/4.0a6/versions-prod.cfg
+extends = https://raw.githubusercontent.com/zopefoundation/Zope/master/versions-prod.cfg
 develop = .
 extensions = mr.developer
 sources = sources
 sources-dir = auto-checkout-sources
 auto-checkout =
-    Zope2
+    Zope
```

Comment:
Is this actually currently needed or is the latest released version enough?

Reply:
I removed the checkout.